### DISCUSSION TITLE
Hide players with $0 from /balancetop

### DISCUSSION THREAD

**Discussion by molor** [EssentialsX/Essentials#4205] [Ideas and Feature Suggestions] [answered]
This is a feature request and it's described in the title.

It would be nice if this setting was in the configuration file, because my server have 13k+ players and ~90 % of them are with zero balance, what makes a lot of useless /baltop pages appear. With this setting, If players with an zero balance were removed BEFORE the sorting was done, it would happen much faster and it would be more convenient...

**Comment by DebugOk** [comment, marked as answer]
I've created a [Pull Request](https://github.com/EssentialsX/Essentials/pull/4226) that adds what you suggested, however it may not be fully optimized yet as I'm not sure how that would be done.

Edit (triagonal): Implemented in above PR, available in dev builds at https://essentialsx.net/downloads.html
